<launch>
  <node name="panda_cmd_server" pkg="rl_interface" type="cmd_service.py" ></node>
</launch>